docs/BR.md

```diff
@@ -1756,153 +1754,1152 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+| `certificatePolicies`      |              |                                    |
+| \ \ **1**                  |              | The first `PolicyInformation` present in the `certificatePolicies`. |
+| \ \ \ \ `policyIdentifier` | MUST         | The `anyPolicy` (OID: 2.5.29.32.0) identifier. |
+| \ \ \ \ `policyQualifiers` | MUST NOT     |                                    |
+| \ \ **2**                  | MUST NOT     | When the `anyPolicy` policy is present, the CA MUST NOT include any further `PolicyInformation` values. |
 
-#### 7.1.2.3 Subscriber Certificate
+Table: Policy Restricted
 
-a. `certificatePolicies`
+| __Field__                  | __Presence__ | __Description__ |
+| ---                        | -            | ------          |
+| `certificatePolicies`      |              |                 |
+| \ \ **1**                  | MUST         | The first `PolicyInformation` present in the `certificatePolicies`. |
+| \ \ \ \ `policyIdentifier` |              | A Reserved Certificate Policy Identifier (see [Section 7.1.6.1](#7161-reserved-certificate-policy-identifiers)). |
+| \ \ \ \ `policyQualifiers` | MUST NOT     |                 |
+| \ \ **2+**                 | SHOULD       | The CA SHOULD include additional Reserved Certificate Policy Identifiers for each type of Subscriber certificate that may be issued beneath it. |
+| \ \ \ \ `policyIdentifier` | MUST         | A Reserved Certificate Policy Identifier (see [Section 7.1.6.1](#7161-reserved-certificate-policy-identifiers)). |
+| \ \ \ \ `policyQualifiers` | MUST NOT     |                 |
+| \ \ **3+**                 | MAY          | The CA MAY include additional `PolicyInformation` values that meet the following profile: |
+| \ \ \ \ `policyIdentifier` | MUST         | An identifier documented by the CA in its Certificate Policy and/or Certification Practice Statement. |
+| \ \ \ \ `policyQualifiers` | MUST NOT     |                 |
+| \ \ **4**                  | MUST NOT     | The CA MUST NOT include any additional `PolicyInformation` values that do not meet the above profile. |
+
+##### 7.1.2.10.5 Certificate Policies - Non-Affiliated CA
+
+The following requirements apply to the Certificate Policies extension within CA certificates that are issued to and operated by a CA that is an not an Affiliate of the Issuing CA.
+
+If present, the Certificate Policies extension MUST be formatted as follows:
+
+| __Field__                  | __Presence__ | __Description__ |
+| ---                        | -            | ------          |
+| `certificatePolicies`      |              |                 |
+| \ \ **1**                  | MUST         | The first `PolicyInformation` present in the `certificatePolicies`. |
+| \ \ \ \ `policyIdentifier` |              | A Reserved Certificate Policy Identifier (see [Section 7.1.6.1](#7161-reserved-certificate-policy-identifiers)). |
+| \ \ \ \ `policyQualifiers` | SHOULD NOT   | See below for restrictions on `policyQualifiers` |
```

Comment:
The first Certificate Policies entry for non-affiliated CAs (7.1.2.10.4) says "MUST NOT" where this is "SHOULD NOT". Is there a reason for the difference? If the intention is to not include a policyQualifier for reserved policy identifiers, should this also be a "MUST NOT"?

Reply:
Thanks for flagging!

Under the existing BRs, the language in 7.1.2.2 states:
```
The following fields MAY be present if the Subordinate CA is not an Affiliate of the
entity that controls the Root CA
```

The key being "is not an Affiliate of the entity that controls the Root CA"

Logically, if we read this as written, then it means that if the Subordinate CA **is** an Affiliate, then those fields MUST NOT be present, because they are only permitted under certain situations.

So the intent here was to transcribe the existing requirement.

Reply:
@chrisbn Did this answer address your concern?

Reply:
@chrisbn I reworked this some in https://github.com/sleevi/cabforum-docs/pull/36/commits/197f0964d991059609aa1acd7cb5eb6a53bfc343 and I'm wondering if this fully addresses your concern?

Namely, this ends up allowing `cPSuri` everywhere for the time being, following the last validation call.